administrator/components/com_localise/language/en-GB/com_localise.ini

```diff
@@ -232,13 +240,20 @@ COM_LOCALISE_TEXT_TRANSLATION_TRANSLATED="Translated"
 COM_LOCALISE_TEXT_TRANSLATION_UNCHANGED="Unchanged"
 COM_LOCALISE_TEXT_TRANSLATION_UNTRANSLATED="Untranslated"
 COM_LOCALISE_TITLE_TRANSLATION="Translation"
-COM_LOCALISE_TOOLTIP_TRANSLATION_INSERT="<strong>Insert</strong><br>Click to insert"
+COM_LOCALISE_TOOLTIP_TRANSLATION_EXTRA_KEYS_IN_TRANSLATION="If you do not want to keep a string in the translation, you should check the checkbox to remove it on 'Save'.<br> Some strings may not be in reference anymore.<br>Some strings may be specific plural forms that you need to keep."
+COM_LOCALISE_TOOLTIP_TRANSLATION_EXTRA_KEYS_IN_TRANSLATION_ICON="Checking the box this string will be removed from your translation."
 COM_LOCALISE_TOOLTIP_TRANSLATION_GOOGLE="<strong>Google</strong><br>Click to find a google translation"
+COM_LOCALISE_TOOLTIP_TRANSLATION_GRAMMAR_CASE="Grammar case\\nIf it is a typing error in the language 'en-GB' if you check the box, this changed text will not be shown in the other 'xx-XX' languages."
+COM_LOCALISE_TOOLTIP_TRANSLATION_INSERT="<strong>Insert</strong><br>Click to insert"
+COM_LOCALISE_TOOLTIP_TRANSLATION_KEY_TO_DELETE="Key to delete the next Joomla release"
+COM_LOCALISE_TOOLTIP_TRANSLATION_KEYS_TO_DELETE="What is shown below are the keys that are going to be removed in the next Joomla release, and not in this moment.<br>It is not recommended to edit or try to delete them because it would have the same effect as applying that action on your current 'en-GB' installed instance."
```

Comment:
Suggest `The strings displayed below will be removed in the next Joomla release.<br>It is not recommended to edit or try to delete them because it would have the same effect as applying that action on your current 'en-GB' installed instance."`

Reply:
Done